kroxylicious-operator/pom.xml

```diff
@@ -322,4 +322,143 @@
             </testResource>
         </testResources>
     </build>
+
+    <profiles>
+        <profile>
+            <id>kubeTest</id>
+            <properties>
+                <container.tool>podman</container.tool> <!-- override for docker -->
+                <container.tag>test</container.tag>
+                <image.archive.file>/tmp/myimage.tar</image.archive.file>
+            </properties>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.codehaus.mojo</groupId>
+                        <artifactId>exec-maven-plugin</artifactId>
+                        <version>3.5.0</version>
+                        <executions>
+                            <execution>
+                                <!-- build the container image -->
+                                <id>podman-build</id>
+                                <goals>
+                                    <goal>exec</goal>
+                                </goals>
+                                <phase>package</phase>
+                                <configuration>
+                                    <workingDirectory>..</workingDirectory>
+                                    <executable>${container.tool}</executable>
+                                    <arguments>
+                                        <argument>build</argument>
+                                        <argument>--file=./Dockerfile.operator</argument>
+                                        <argument>--build-arg=KROXYLICIOUS_VERSION=${project.version}</argument>
+                                        <argument>--tag=kroxylicious.io/operator:${container.tag}</argument>
+                                    </arguments>
+                                </configuration>
+                            </execution>
+                            <execution>
+                                <!-- export the container image to a temp file -->
+                                <id>podman-push</id>
+                                <goals>
+                                    <goal>exec</goal>
+                                </goals>
+                                <phase>package</phase>
+                                <configuration>
+                                    <workingDirectory>..</workingDirectory>
+                                    <executable>${container.tool}</executable>
+                                    <arguments>
+                                        <argument>push</argument>
+                                        <argument>kroxylicious.io/operator:${container.tag}</argument>
+                                        <argument>docker-archive:${image.archive.file}</argument>
```

Comment:
`oci-archive` didn't work for me when I tried to `minikube image load` it, so I went with `docker-archive`.

Annoyingly podman barfs if this file already exists, so I might need an exec of `rm` to remove it :-(